_posts/blog/2017-07-19-Actions-for-android-devs.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Google Assistant Development for Android Devs (Part 1)"
+author: Patrick Jackson
+categories: Development
+tags: [Google Assistant, Actions on Google, conversational interfaces, Android, Kotlin]
+---
+
+
+# Google Assistant Development for Android Devs (Part 1)
+
+![](https://developers.google.com/actions/images/surfaces-hero.svg)
```

Comment:
We have it set up to pull from the `featured-image` front matter variable. There's an example in the [readme](https://github.com/TicketmasterMobileStudio/ticketmastermobilestudio.github.io#featured-image).